rchain/crypto.py

```diff
@@ -52,6 +52,9 @@ def get_rev_address(self) -> str:
     def get_eth_address(self) -> str:
         return keccak(self.to_bytes()[1:]).hex()[-40:]
 
+    def __hash__(self):
```

Comment:
[You also need to define `__eq__()`](https://docs.python.org/3/reference/datamodel.html?highlight=__hash__#object.__hash__):
`If a class does not define an __eq__() method it should not define a __hash__() operation either; `